aya/src/maps/perf/perf_buffer.rs

```diff
@@ -273,14 +273,19 @@ impl Drop for PerfBuffer {
     }
 }
 
+#[cfg(target_pointer_width = "32")]
+type OffsetType = i32;
+#[cfg(target_pointer_width = "64")]
+type OffsetType = i64;
```

Comment:
I think that this should be `libc::off_t`?

Reply:
it was declared as i64 and i've put the conditional definition, i can change it to libc::off_t

Reply:
Yes please - here and in the other cases you found I was sloppy and used rust primitive types where I should have used libc types

Reply:
fair enough, will do asap :)